samcli/commands/validate/validate.py

```diff
@@ -170,20 +199,97 @@ def _lint(ctx: Context, template: str, template_path: str) -> None:
+                fg="red",
+            )
+            raise UserException(
+                "Serverless Rules package (cfn-lint-serverless) is not installed. "
+                "Please install it using: pip install cfn-lint-serverless"
+            )
+            
+        try:
+            # Try to import the package
+            import cfn_lint_serverless
+            print("Debug info: cfn_lint_serverless package import successful")
+            
+            # Add Serverless Rules to the rule list
+            rules_to_append.append("cfn_lint_serverless.rules")
+            click.secho("Serverless Rules enabled for linting", fg="green")
+        except ImportError as e:
+            print(f"Debug info: cfn_lint_serverless import error = {e}")
+            click.secho(
+                "Serverless Rules package (cfn-lint-serverless) is not installed. "
+                "Please install it using: pip install cfn-lint-serverless",
+                fg="red",
+            )
+            raise UserException(
+                "Serverless Rules package (cfn-lint-serverless) is not installed. "
+                "Please install it using: pip install cfn-lint-serverless"
+            )
+    
+    # Support for the new extra_lint_rules option
+    if extra_lint_rules:
+        print(f"Debug info: extra_lint_rules option is activated. Value: {extra_lint_rules}")
+        # Track usage of Extra Lint Rules
+        EventTracker.track_event("UsedFeature", "ExtraLintRules")
+        
+        # Parse comma-separated rule modules
+        modules = [module.strip() for module in extra_lint_rules.split(',') if module.strip()]
+        print(f"Debug info: parsed rule modules list = {modules}")
+        
+        # Add each module to the rule list
+        rules_to_append.extend(modules)
+        click.secho(f"Extra lint rules enabled: {extra_lint_rules}", fg="green")
+    
+    # Add rules to linter_config if any exist
+    if rules_to_append:
+        print(f"Debug info: rules to append = {rules_to_append}")
+        linter_config["append_rules"] = rules_to_append
+        print(f"Debug info: updated linter_config = {linter_config}")
+
     config = ManualArgs(**linter_config)
+    print(f"Debug info: config creation completed")
 
     try:
+        print(f"Debug info: starting lint function call")
         matches = lint(template, config=config)
+        print(f"Debug info: lint function call completed, matches = {matches}")
     except InvalidRegionException as ex:
+        print(f"Debug info: InvalidRegionException occurred = {ex}")
         raise UserException(
             f"AWS Region was not found. Please configure your region through the --region option.\n{ex}",
             wrapped_from=ex.__class__.__name__,
         ) from ex
+    except Exception as e:
+        print(f"Debug info: exception occurred = {e}")
+        raise
 
     if not matches:
+        print(f"Debug info: template validation successful")
```

Comment:
I don't think we need all these debugging messages though. Some make sense, but this part is already printing something else already.